shared/common-adapters/confirm-modal/index.js

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react'
+import {Box, Box2, ButtonBar, HeaderOrPopup, Icon, Text, WaitingButton} from '..'
+import * as Styles from '../../styles'
+import type {IconType} from '../icon.constants'
+
+// generally one of icon or header will be given
+export type Props = {|
+  confirmText?: string,
+  content?: React.Node,
+  description: string,
+  header?: React.Node,
+  icon?: IconType,
+  onCancel: () => void,
+  onConfirm: () => void,
+  prompt: string,
+  waitingKey?: string,
+|}
+
+class _ConfirmModal extends React.Component<Props> {
+  render() {
+    return (
+      <Box style={styles.mobileFlex}>
+        <Box2 direction="vertical" style={styles.container}>
+          <Box2
+            alignItems="center"
+            direction="vertical"
+            fullWidth={true}
+            fullHeight={true}
+            style={styles.inner}
+          >
+            {this.props.icon && (
+              <Icon
+                boxStyle={styles.icon}
+                color={Styles.globalColors.black_50}
+                fontSize={Styles.isMobile ? 64 : 48}
+                style={styles.icon}
+                type={this.props.icon}
+              />
+            )}
+            {this.props.header && (
+              <Box2 alignItems="center" direction="vertical" style={styles.icon}>
+                {this.props.header}
+              </Box2>
+            )}
+            <Text center={true} style={styles.text} type="HeaderBig">
+              {this.props.prompt}
+            </Text>
+            <Text center={true} style={styles.text} type="Body">
+              {this.props.description}
+            </Text>
+            {this.props.content}
+          </Box2>
+        </Box2>
+        <Box2 direction="horizontal" style={styles.buttonBox}>
+          <ButtonBar direction="row" fullWidth={true} style={styles.buttonBar}>
+            {!Styles.isMobile && (
+              <WaitingButton
+                fullWidth={true}
+                type="Secondary"
+                label="Cancel"
+                onClick={this.props.onCancel}
+                waitingKey={this.props.waitingKey}
+              />
+            )}
+            <WaitingButton
+              fullWidth={true}
+              type="Danger"
+              label={this.props.confirmText || 'Confirm'}
+              onClick={this.props.onConfirm}
+              waitingKey={this.props.waitingKey}
+            />
+          </ButtonBar>
+        </Box2>
+      </Box>
+    )
+  }
+}
+const ConfirmModal = HeaderOrPopup(_ConfirmModal)
+
+const styles = Styles.styleSheetCreate({
+  buttonBar: Styles.platformStyles({
+    isElectron: {
+      minHeight: 40,
+    },
+  }),
+  buttonBox: Styles.platformStyles({
+    common: {
+      backgroundColor: Styles.globalColors.white,
+      padding: Styles.globalMargins.xsmall,
+    },
+    isElectron: {
+      borderRadius: 4,
+    },
+  }),
+  container: Styles.platformStyles({
+    common: {
+      backgroundColor: Styles.globalColors.yellow3,
```

Comment:
Is this really supposed to be yellow or is it just guideline color in the design mock up?

Reply:
are you suggesting the yellow isn't supposed to be yellow? 
https://app.zeplin.io/project/56a15d74e1e098dd3116d10b/screen/5c75bc9426c62e6244ba583e

Reply:
It's not meant to be yellow. The yellow is used to highlight the margins/padding.

Reply:
ok now it's not yellow anymore

Reply:
:white_check_mark: